### PR TITLE
Add responses to diagnostic

### PR DESCRIPTION
**Pull request by akgoode**
Comfort: 5
Clarity: 5

I referenced the clinic code along for the exact syntax for models and serializers.  I also googled dependent destroy.

### CODE REVIEW

**Review by BenGitsCode — COMMENTED** [review]
Please softwrap your responses at 80 characters.

**Comment by akgoode** [comment]
Fixed

File: diagnostic.md [hunk 1]
```diff
@@ -40,31 +52,46 @@ like to show all movies favorited by a profile on
 `http://localhost:3000/profiles/1`
 
   ```md
-    # < Your Response Here >
+    The purpose of a serializer is to take the requested data from the database
+    and turn it into a json string.
```
Comment (BenGitsCode) [inline diagnostic.md:56]: Yes but an important part is that it determines how the string is returned; it filters the JSON so we can control the what data is sent back from the server.

**Review by BenGitsCode — COMMENTED** [review]
Thanks for fixing the wrap, you didn't need to right away but I appreciate the extra readability. 

Great job here! 5/5

**Pull request closed without merging**